typings/index.d.ts

```diff
@@ -4,11 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/**
- * @fileoverview Type definitions for Blockly.
- * @author samelh@google.com (Sam El-Husseini)
- */
-
-export * from './core/blockly';
-export * as libraryBlocks from './blocks';  // Handcrafted file.
+import * as core from './core/blockly';
+import * as libraryBlocks from './blocks';  // Handcrafted file.
+export {core};
```

Comment:
note to self: need to double check that these two sides are equivalent and i haven't yet

Reply:
This wasn't quite right since previously core wasn't `export * as` so these were not equivalent before and after. Now they are and I tested using blockly-typescript-libcheck.